array/src/main/java/ArrayExample.java

```diff
@@ -0,0 +1,12 @@
+public class ArrayExample {
+
+    public static String[] getStudents(int size) {
+        String[] studentArray = new String[size];
+
+        for (int i = 0; i < studentArray.length; i++) {
+            studentArray[i] = "Student " + (i + 1);
+        }
+        return studentArray;
+    }
```

Comment:
```suggestion
    public static String[] createStudentsArray(int size) {
        String[] students = new String[size];

        for (int i = 0; i < students.length; i++) {
            students[i] = "Student " + (i + 1);
        }
        return students;
    }
```
This doesn't really get something from an existing resource, maybe rename it, not sure. Adding the type to the name of a variable is not necessay, imho.